index.html

```diff
@@ -0,0 +1,300 @@
+                        </div>
+                    </div>
+                    <div class="card">
+                        <div class="card-body">
+                            <h5>
+                                Multi-Post Stories<br>
+                                Gain+Glory
+                            </h5>
+                            <div class="work-card-list">
+                                <ul class="list-view">
+                                    <li class="list-item">Ruby on rails</li>
+                                    <li class="list-item">css</li>
+                                    <li class="list-item">JavaScript</li>
+                                    <li class="list-item">html</li>
+                                </ul>
+                            </div>
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                    <div class="card">
+                        <div class="card-body">
+                            <h5>
+                                Multi-Post Stories<br>
+                                Gain+Glory
+                            </h5>
+                            <div class="work-card-list">
+                                <ul class="list-view">
+                                    <li class="list-item">Ruby on rails</li>
+                                    <li class="list-item">css</li>
+                                    <li class="list-item">JavaScript</li>
+                                    <li class="list-item">html</li>
+                                </ul>
+                            </div>
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                    <div class="card">
+                        <div class="card-body">
+                            <h5>
+                                Multi-Post Stories<br>
+                                Gain+Glory
+                            </h5>
+                            <div class="work-card-list">
+                                <ul class="list-view">
+                                    <li class="list-item">Ruby on rails</li>
+                                    <li class="list-item">css</li>
+                                    <li class="list-item">JavaScript</li>
+                                    <li class="list-item">html</li>
+                                </ul>
+                            </div>
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+            </section>
+
+            <!-- About me section -->
+            <div class="about-me main-l-item">
+                <section class="about-me-describe">
+                    <h3>
+                        About me
+                    </h3>
```

Comment:
Please ensure that the main container of the `About me` is a ``section`` tagaccording to the project requirement. At the moment you have `div` tag and  a `section` tag inside the `div` tag. Please you can do it the other way round. You can have a `section` tag and the main container and then a `div` tag inside the `section` tag.

Reply:
The problem here is that whenever I use `section` without a direct `h` element, I get warnings while linting.
I'm not sure if it's acceptable to have warning, so that's why I made it that way.

Reply:
Fixed